omnibus/config/projects/agent.rb

```diff
@@ -26,7 +26,7 @@
 else
   COMPRESSION_THREADS = 1
 end
-if ENV.has_key?("DEPLOY_AGENT") && ENV["DEPLOY_AGENT"] == "true"
+if ENV.has_key?("DEPLOY_AGENT") && ENV["DEPLOY_AGENT"] == "true" && ENV['BUCKET_BRANCH'] != "nightly"
```

Comment:
Could you add a comment to clarify the intent?

Reply:
Oh wait, actually how does this behave when `BUCKET_BRANCH` isn't set, as is the case for local builds?